liota/lib/transports/web_socket.py

```diff
@@ -56,6 +56,7 @@ def connect_soc(self):
             self.WebSocketConnection(self.url, False)
         except Exception:
             log.exception("WebSocket exception, please check the WebSocket address and try again.")
+            raise Exception("WebSocket exception, please check the WebSocket address and try again.")
```

Comment:
Over here exception might be of wide range and we need to catch all of them ranging from websocket object none to websocket close to websocket send exceptions. Thus, it is fine to catch all type of exceptions in typical scenarios. 

http://stackoverflow.com/questions/21938/is-it-really-that-bad-to-catch-a-general-exception